src/services/Look/index.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+import { IncomingMessage } from 'http';
+
+import { listConfig, partialUpdateConfig, deleteConfig } from './config';
+import { ILook, ILookInputDTO } from '@src/interfaces';
+
+const list = async (req?: IncomingMessage): Promise<ILook[]> =>
+  axios(listConfig(req)).then((res) => res.data);
+
+const partialUpdate = async (
+  id: number,
+  updateDTO: ILookInputDTO,
+  req?: IncomingMessage
+): Promise<ILook> =>
+  axios(partialUpdateConfig(id, updateDTO, req)).then((res) => res.data);
+
+const _delete = async (id: number, req?: IncomingMessage): Promise<ILook> =>
+  axios(deleteConfig(id, req)).then((res) => res.data);
+
+const LookService = {
+  list,
+  partialUpdate,
+  _delete,
```

Comment:
```suggestion
  delete: _delete,
```

delete가 예약어여서 변수 이름으로 못 쓰지만, property name으론 상관 없기 때문에 다시 바꿔서 써줍니당

Reply:
3978588